__tests__/helpers/openingHoursHelper.test.js

```diff
@@ -0,0 +1,215 @@
+import { getReadableDay, isOpen } from '../../src/helpers';
+
+const todayString = getReadableDay(new Date());
+
+const FAKE_NOW_DATE = new Date();
+FAKE_NOW_DATE.setHours(12);
+FAKE_NOW_DATE.setMinutes(0);
+FAKE_NOW_DATE.setSeconds(0);
+FAKE_NOW_DATE.setMilliseconds(0);
+
+const isOpenWithFakeTimeDateTime = (openingHours) => isOpen(openingHours, FAKE_NOW_DATE);
+
+describe('testing the opening times handling', () => {
+  test('single opening intervals work as expected', () => {
+    expect(
+      isOpenWithFakeTimeDateTime([
+        {
+          open: true,
+          timeFrom: '16:00',
+          timeTo: '17:00',
+          weekday: todayString
+        }
+      ]).open
+    ).toEqual(false);
+    expect(
+      isOpenWithFakeTimeDateTime([
+        {
+          open: true,
+          timeFrom: '10:00',
+          timeTo: '17:00',
+          weekday: todayString
+        }
+      ])
+    ).toEqual({ open: true, timeDiff: 300 });
```

Comment:
this first test block contains just "....open" checks except of this one. wouldn't it be better to also check .open equals true here and ignore the `timeDiff` with reason?